app/models/piece.rb

```diff
@@ -2,75 +2,62 @@
 
 class Piece < ApplicationRecord
   belongs_to :game
+  
+  def is_white?
+    piece_number < 6
+  end
+
+  def is_black?
```

Comment:
I think we only need to have one of these methods. Either is_white? or is_black? Because if is_white? returns false, we know the piece is black.

Reply:
Good point. I like the two for clarity, but probably better as one.